package.json

```diff
@@ -38,6 +39,7 @@
   },
   "devDependencies": {
     "ava": "*",
+    "typescript": "^3.0.1",
```

Comment:
TS is not necessary for development. Remove this dependency and `tsconfig.json`.